templates/Web/Projects/AddPackageJsonFullStackDefault/.template.config/template.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "author": "Microsoft",
+  "classifications": ["Universal"],
+  "name": "AddPackageJsonFullStackDefault",
+  "groupIdentity": "wts.Proj.AddPackageJsonFullStackDefault",
+  "identity": "wts.Proj.AddPackageJsonFullStackDefault",
+  "description": "",
+  "tags": {
+    "language": "Any",
+    "type": "project",
+    "wts.type": "project",
+    "wts.projecttype": "FullStackWebApp",
+    "wts.frontendframework": "all",
+    "wts.backendframework": "all",
+    "wts.platform": "Web",
+    "wts.version": "1.0.0",
+    "wts.displayOrder": "1"
+  },
+  "sourceName": "wts.AddPackageJsonFullStackDefault",
+  "preferNameDirectory": true,
+  "PrimaryOutputs": [],
+  "symbols": {
+  },
+  "postActions": [
+    {
+      "description": "Add Base Start Scripts",
+      "manualInstructions": [],
+      "actionId": "CB387AC0-16D0-4E07-B41A-F1EA616A7CA9",
```

Comment:
what is the action id? is it save to put it inside GitHub?

Reply:
Completely safe. This id is used by corets to do the right custom post action. It is specifying that i want to do a json addition. Thanks for the concern but no it is not a private key haha, has nothing to do with access or anything. Just the unique identifier for the post action we want. There's a whole list on corets for the custom postactions.

Reply:
thank you for the explanation. I will approve the PR